src/components/learner-credit-management/cards/NewAssignmentModalButton.jsx

```diff
@@ -147,7 +147,7 @@ const NewAssignmentModalButton = ({ enterpriseId, course, children }) => {
         });
 
         // Navigate to the activity tab
-        history.push(pathToActivityTab);
+        navigate(pathToActivityTab);
```

Comment:
Noticed a bug with this implementation where the url is not being changed to reflect the activity tab and is not redirecting correctly.  (See video)

The first half of the video is the PR, while the second half is the current stage environment.
https://github.com/openedx/frontend-app-admin-portal/assets/82611798/3e4568f3-5ecd-4501-aa5e-69e11f611194